open-authoring/open-authoring/OpenAuthoringProvider.tsx

```diff
@@ -113,55 +101,83 @@ export const OpenAuthoringProvider = ({
   )
 }
 
-const OpenAuthoringAuthModal = ({ onUpdateAuthState, close }) => {
-  let modalProps
+const OpenAuthoringAuthModal = ({ close }) => {
+  const [modalProps, setModalProps] = useState({
```

Comment:
It feels a bit weird to me to have the loading state be using these modal props.